src/Services/ItemService.cs

```diff
@@ -42,13 +42,21 @@ public class ItemService {
         }
 
         public int GetItemQuantity(ItemDataRelationship itemData) {
+            return GetItemQuantityImpl(itemData);
+        }
+
+        public int GetBulkItemQuantity(ItemDataRelationship itemData, int quantity) {
+            return GetItemQuantityImpl(itemData, quantity);
+        }
+
+        private int GetItemQuantityImpl(ItemDataRelationship itemData, int repeat = 1) {
```

Comment:
maybe just `GetItemQuantity(ItemDataRelationship itemData, int bulkSize = 1)` instead of 3 functions

Reply:
Adjusted